web/src/pages/Cases/CaseDetails/Appeal/Classic/StageExplainer.tsx

```
@@ -41,46 +43,42 @@ interface IStageExplainer {
-        If the current winner is not fully funded in time, the option funded during the previous stage will be declared
-        as the final winner.
-      </p>
-      <p>
-        {" "}
-        Following choices were funded in the stage 1 :{" "}
-        <small>{options?.map((option) => (option?.funded ? option.title : null))}</small>
-      </p>
+      <p dangerouslySetInnerHTML={{ __html: t("appeal.stage_two_explanation_1") }} />
+      <p dangerouslySetInnerHTML={{ __html: t("appeal.stage_two_explanation_2") }} />
+      <p
+        dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)